### PR TITLE
[14.0][MIG]purchase_supplier_rank: Migration to 14.0

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
Standard migration
@ForgeFlow

### CODE REVIEW

File: purchase_supplier_rank/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+from . import models
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    partners = env["purchase.order"].search([]).mapped("partner_id")
+    partners |= partners.mapped("commercial_partner_id")
+    partners._increase_rank("supplier_rank")
```
Comment (ChrisOForgeFlow) [inline purchase_supplier_rank/__init__.py]: You could update the ranking in a more specific way, increasing the number of purchases that the supplier has, the function accepts a parameter n to be able to specify it

https://github.com/odoo/odoo/blob/d80ec72696010d668047c8fb1d9665f6aa1b2ee6/addons/account/models/partner.py#L495

**Review by ChrisOForgeFlow — CHANGES_REQUESTED** [review]

File: purchase_supplier_rank/__init__.py (hunk 1 above)
Comment (GuillemCForgeFlow) [reply purchase_supplier_rank/__init__.py]: Thanks for the reply! Hope that works! @ChrisOForgeFlow

**Review by ChrisOForgeFlow — APPROVED** [review]
+1

**Comment by SerM11** [comment]
We want to test this pr but runbot/runboat is not working.

**Comment by GuillemCForgeFlow** [comment]
> We want to test this pr but runbot/runboat is not working.

Should be working now! @SerM11

**Pull request closed without merging**